.github/workflows/generate_library.yml

```diff
@@ -38,11 +38,13 @@ jobs:
         cache: maven
 
     - name: "Clean library folder"
-      run: find . ! -name 'poetry.lock' -type f -exec rm -f {} +
+      run: find . ! -name 'poetry.lock' -path './src' -prune -o -type f -exec rm -f {} +
```

Comment:
Could we just use maven clean to do all of this and remove this step entirely? It's a bit odd having two different ways of doing different parts of the same job.